app/views/permissions/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class='container mt-5'>
+	<div class='mb-3'><b>Email : </b><%= @user.email%></div>
+	<div class='mb-3'><b>Username : </b><%= @user.username%></div>
+	<div class='mb-3'><b>Role : </b>
+		<% if @user.roles.first.present? %>
+			<%= @user.roles.last.name%><br>
+		<%else%>
+			no role assigned<br>
+		<%end%>
+	</div>
+	
+	<%= select_tag(:role, 
+	options_for_select(['admin','moderator','manager','viewer'],
+	if @user.roles.first.present?
+		@user.roles.last.name 
+	end), :id=>'select', :user_id=>@user.id) %>	
+	<%= button_tag "remove role", :id=>"rmrole", :user_id=>@user.id, :class=>'btn btn-danger btn-sm'%><br>
+	<%= link_to 'Back', permissions_path, :class=>"btn btn-dark btn-sm mt-4" %>
+</div>
+
+
+<script>
+	$(document).ready(function(){
+		$("#select").change(function(){
```

Comment:
Good work, But it/class name be more specific

Reply:
done